inst/www/shared/shiny.css

```diff
@@ -12,6 +12,11 @@ pre.shiny-text-output.noplaceholder:empty {
   height: 0;
 }
 
+.shiny-image-output img, .shiny-plot-output img {
```

Comment:
Change to something like `.shiny-image-output img.scalable`, and set the class on img tag.